src/routes/Facility.js

```diff
@@ -52,4 +52,51 @@ router.post('/addNew',_validateFacility,addFacility);
  *         description: Server Err
  */
 
+router.put('/:id',_validateFacility,updateFacility);
+/**
+ * @swagger
+ * /facility:
+ *   put:
+ *     summary: update facility 
+ *     tags: [facility]
+ *     description: update facility
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: location
+ *         required: true
+ *         type: string
+ *       - name: address
+ *         required: true
+ *         type: string
+ *       - name: number of rooms
+ *       - name: room detail
+ *       - name: images
+ *         required: true
+ *         type: string
+ *     responses:
+ *       200:
+ *         description: facility addedd Successfully 
+ *       500:
+ *         description: Server Err
+ */
```

Comment:
Kindly revisit the `PUT` and `POST` methods are not well documented..... These endpoints are not working on swagger.